src/src/LocalTests/PrepareQMLog.php

```diff
@@ -179,6 +181,25 @@ public function summarize_qm_logs( string $directory ): array {
 		foreach ( $data as $hash => $logs ) {
 			foreach ( $logs as $type => $type_logs ) {
 				foreach ( $type_logs as $info ) {
+					// Ignore some compatbility extension set issues.
+
+					// Ignore a deprecation warning from Jetpack about itself.
+					$is_jetpack_geo_deprecation = stripos( $info['message'], 'Class Jetpack_Geo_Location is' ) !== false;
```

Comment:
Do we want to ignore this if we are testing jetpack?

Reply:
IMO yes, they're doing it to themselves. Also it'll get removed at some point, depending on what release the PR makes it into.